components/post_view/post_image/post_image.tsx

```diff
@@ -1,29 +1,36 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 
 import ExternalImage from 'components/external_image';
 import SizeAwareImage from 'components/size_aware_image';
 import ViewImageModal from 'components/view_image';
 
-export default class PostImage extends React.PureComponent {
-    static propTypes = {
-        imageMetadata: PropTypes.object.isRequired,
-        link: PropTypes.string.isRequired,
-        post: PropTypes.object.isRequired,
-    }
+type Props = {
+    imageMetadata: {
+        height?: number,
+        width?: number
+        format?: string
+    },
+    link: string,
+    post: any,
```

Comment:
Please instead of any use `Post` as you can see [here](https://github.com/mattermost/mattermost-webapp/blob/master/components/post_view/post_body_additional_content/post_body_additional_content.tsx#L18)